app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -31,8 +31,6 @@
         document.getElementById("state_file_nj_tenant_eligibility_form_tenant_home_subject_to_property_taxes"),
         document.getElementById("state_file_nj_tenant_eligibility_form_tenant_building_multi_unit"),
         document.getElementById("state_file_nj_tenant_eligibility_form_tenant_access_kitchen_bath"),
-        document.getElementById("state_file_nj_tenant_eligibility_form_tenant_more_than_one_main_home_in_nj"),
-        document.getElementById("state_file_nj_tenant_eligibility_form_tenant_shared_rent_not_spouse")
```

Comment:
[dust] I know Tristan wants to test & experiment with this feature with the possibility of rolling back if it makes things more complicated for people. So because of that, I don't think we should include a database migration in *this* PR that removes these fields from the database altogether in case of that situation. That said, I do think we should ensure a ticket gets put in the backlog to do so in the future, because if these fields are never being used, they *should* be removed from the database and keep a clean data model